site/index.html

```diff
@@ -5,37 +5,48 @@
 ---
 <div class="header">
    <div class=" container ">
-      <div class=" row ">
-         <div class=" span12 col-sm-12 hidden-xs"><br></div>
+      <div class="row" >
+         <div class=" span12 col-sm-12 hidden-xs"><br><br></div>
+      </div>
+      <div class=" row " style="display: flex; align-items: center">
          <div class=" span6 col-md-6">
             <div class="branding">
-               <br class="hidden-xs"><img src="/img/haskell-logo.svg" class="img-responsive">
-               <h4 class="summary">An advanced, purely functional programming language</h4>
+              <br class="hidden-xs">
+              <img src="/img/haskell-logo.svg" class="img-responsive">
+              <h4 class="summary">An advanced, purely functional programming language</h4>
             </div>
          </div>
          <div class=" span6 col-md-6">
-            <div class="branding sample">
-               <br class="visible-xs visible-sm">
-               <h4 class="tag">Declarative, statically typed code.</h4>
-               <div title="This example is contrived in order to demonstrate what Haskell looks like, including: (1) where syntax, (2) enumeration syntax, (3) pattern matching, (4) consing as an operator, (5) list comprehensions, (6) infix functions. Don&#39;t take it seriously as an efficient prime number generator." class="code-sample">
-                  <pre><span class='hs-definition'>primes</span> <span class='hs-keyglyph'>=</span> <span class='hs-varid'>filterPrime</span> <span class='hs-keyglyph'>[</span><span class='hs-num'>2</span><span class='hs-keyglyph'>..</span><span class='hs-keyglyph'>]</span>
+            <div class="row" id="code-example">
+               <div class="branding sample">
+                  <br class="visible-xs visible-sm">
+                  <h4 class="tag">Declarative, statically typed code.</h4>
+                  <div title="This example is contrived in order to demonstrate what Haskell looks like, including: (1) where syntax, (2) enumeration syntax, (3) pattern matching, (4) consing as an operator, (5) list comprehensions, (6) infix functions. Don&#39;t take it seriously as an efficient prime number generator." class="code-sample">
+                     <pre><span class='hs-definition'>primes</span> <span class='hs-keyglyph'>=</span> <span class='hs-varid'>filterPrime</span> <span class='hs-keyglyph'>[</span><span class='hs-num'>2</span><span class='hs-keyglyph'>..</span><span class='hs-keyglyph'>]</span>
```

Comment:
The only thing I did here was wrapped all this into a div that has Boostrap class "row".
I need to do this in order to later add a get-started button below it, which is also in its own "row". This is how it is done in Bootstrap.
I didn't touch any of the existing html here, git is showing it as a change only because of the indentation.